checkov/main.py

```diff
@@ -415,6 +425,40 @@ def run(banner: str = checkov_banner, argv: List[str] = sys.argv[1:]) -> Optiona
     return None
 
 
+def report_contributor_metrics(repository: str) -> None:  # ignore: type
+
+    def _parse_gitlog() -> dict[str, Any] | None:
+        process = subprocess.Popen(['git', 'shortlog', '-ne', '--all', '--since', '"90 days ago"', '--pretty="commit-%ct"', '--reverse'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)  # nosec
+        out, err = process.communicate()
+        if err:
+            logger.info('Failed to collect contributor metrics')
+            return None
+        # split per contributor
+        list_of_contributors = out.decode('utf-8').split('\n\n')
```

Comment:
it would nice to move this into a normal function, then it easy to mock it and then write some UTs with expected output and what should come out at the end.

Reply:
Done